File: Doodream/JadenCase 문자열 만들기.js
function solution(s) {
    s = s.toLowerCase();
    s = s.split(" ");


    for (let i = 0; i < s.length; i++) {
        var firstAsciNum = s[i].charCodeAt(0);
        if (firstAsciNum >= 97 && firstAsciNum <= 122) {
            s[i] = s[i].split('');
            s[i][0] = String.fromCharCode(firstAsciNum - 32);
            s[i] = s[i].join('');
        }
    }

    var answer = s.join(' ');
    return answer;
}

var s = "3people unFollowed me";
console.log(solution(s));

File: yeonkyung/JadenCase 문자열 만들기.py
def solution(s):
    answer = []
    i = 0
    while i <= len(s) - 1:
        ch = s[i]
        if i == 0:
            ch = ch.upper()
        else:
            if s[i-1] == ' ':
                ch = ch.upper()
            else:
                ch = ch.lower()
        answer.append(ch)
        i += 1
    return ''.join(answer)

if __name__ == "__main__":
    s = "aa  bb"
    print(solution(s))
